neo/IO/Actors/PriorityMessageQueue.cs

```diff
@@ -30,6 +32,8 @@ public void CleanUp(IActorRef owner, IMessageQueue deadletters)
 
         public void Enqueue(IActorRef receiver, Envelope envelope)
         {
+            Interlocked.Increment(ref idle);
```

Comment:
Nice, so this will ensure idle message is followed by another idle message if nothing else is going on, right?

Actually, won’t this cause an endless loop of idle messages wasting cpu when there is nothing to do?

Reply:
If you want another `Idle` message, just send it to the actor. There can be at most one `Idle` message in the message queue and it is always received last.

Reply:
Makes sense.